app/controllers/plans_controller.rb

```diff
@@ -18,7 +18,7 @@ def new
     authorize @plan
     
     # Get all of the available funders and non-funder orgs
-    @funders = Org.funders.sort{|x,y| x.name <=> y.name }
+    @funders = Org.funders.joins(:templates).where(templates: {published: true}).uniq.sort{|x,y| x.name <=> y.name }
```

Comment:
Ugly syntax but only generates 1 SQL query instead of looping through orgs/templates